### ISSUE TITLE
Pod install fails

### ISSUE THREAD

**Issue by agenthunt** [bugfender/BugfenderSDK-iOS#23]
Hi, I am trying to include the library using cocoapods and it fails with the following error
The 'Pods-BugFenderExample' target has libraries with conflicting names: libbugfendersdk.a

My pod file looks like

```
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'BugFenderExample' do
  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
  # use_frameworks!

  # Pods for BugFenderExample

  pod 'BugfenderSDK', '~> 1.4'

  target 'BugFenderExampleTests' do
    inherit! :search_paths
    # Pods for testing
  end

end

target 'BugFenderExample-tvOS' do
  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
  # use_frameworks!

  # Pods for BugFenderExample-tvOS

  target 'BugFenderExample-tvOSTests' do
    inherit! :search_paths
    # Pods for testing
  end

end
```

Any thoughts?

**Comment by jgimenez** [comment]
Hi @agenthunt, we're working on a new version of the Podspec that will solve this. In the meantime, you can use frameworks and it will work properly. Just uncomment `use_frameworks!` for that.

**Comment by agenthunt** [comment]
Thanks.

**Comment by jgimenez** [comment]
Fixed in 2ed6c5f5287b1d9d1528b1e0f04b12ec82323c77
